src/server/main_service.cc

```diff
@@ -1026,6 +1024,8 @@ bool Service::InvokeCmd(const CommandId* cid, CmdArgList tail_args, ConnectionCo
     DispatchMonitor(cntx, cid, tail_args);
   }
 
+  etl.RecordCmd();
```

Comment:
what's the impact and reasons of this change?

Reply:
This just computes the QPS stats. Not all commands are run via DispatchCommand, but I can move it back

Reply:
Done